src/Components/Components/src/Binding/ModelBindingContext.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.Binding;
+
+/// <summary>
+/// The binding context associated with a given model binding operation.
+/// </summary>
+public class ModelBindingContext
+{
+    // Default binder
+    // Name = URL.Path
```

Comment:
Why use URL.Path and not something like `string.Empty` for the root one? I'm sure you have some reason but it's not obvious to me :)

Reply:
This is using `string.Empty` now